app/models/work.rb

```diff
@@ -0,0 +1,34 @@
+class Work < ApplicationRecord
+  validates :title, presence: true, uniqueness: true
+  has_many :votes
+  has_many :users, through: :votes
+
+  def vote_counter
+    return self.votes.length
+  end
+
+  def self.spotlight_work(works)
+    spotlight_work = works.first
+    works.each do |work|
+      if work.vote_counter > spotlight_work.vote_counter
+        spotlight_work = work
+      end
+    end
```

Comment:
Hm, instead of iterating through this... Consider using an Enumerable method, like `max_by`.
Consider the following refactor, which passes your tests:
```ruby
  def self.spotlight_work(works)
    spotlight_work = works.max_by do |work|
      work.vote_counter
    end
    return nil if spotlight_work.vote_counter == 0
    return spotlight_work
  end
```